lib/parser/openmp-grammar.h

```diff
@@ -233,8 +233,8 @@ TYPE_PARSER("DEFAULTMAP" >>
         construct<OmpClause>(parenthesized(Parser<OmpScheduleClause>{})))
 
 // [Clause, [Clause], ...]
-TYPE_PARSER(
```

Comment:
The copyright at the top of this file shows 2018.  I'm pretty sure that the whole file was created in 2019.

Reply:
I will add that, thanks.

Reply:
It was created in 2018 so the copyright should say 2018-2019.